demo/java/src/ImageClassification.java

```diff
@@ -26,11 +32,19 @@ public static void main(String[] args) {
 
         try {
             classifier = new Classifier(modelPath, deviceName, 0);
+            if (classifier == -1) {
```

Comment:
Compiling error. I think this show be added in java API instead of demo. Other demos have the same issue, too